src/async_hta_service.hpp

```diff
@@ -71,6 +71,28 @@ struct TimeValue
     hta::TimeValue htv;
 };
 
+struct LoggingConfig
+{
+    LoggingConfig() = default;
+
+    LoggingConfig(metricq::json& config)
+    {
+        try
+        {
+            auto logging = config.at("logging");
+            nan_values = logging.value<bool>("nan_values", true);
+            non_monotonic_values = logging.value<bool>("non_monotonic_values", true);
```

Comment:
```suggestion
            nan_values = logging.value<bool>("nan_values", nan_values);
            non_monotonic_values = logging.value<bool>("non_monotonic_values", non_monotonic_values);
```
Removes redundant defaults, should be default-initialized at this point.